main.nf

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env nextflow
-
 /*
 ========================================================================================
```

Comment:
Would be nice to simplify the header here in line with the current template:
https://github.com/nf-core/tools/blob/4ab80b83cd8b6562a88bb38b6aa0a3d082fd4de6/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/main.nf#L3-L10

Reply:
Could also move the `Pipeline overview` to the main `README.md` with links to the relevant tools. See https://github.com/nf-core/chipseq/blob/master/README.md

Reply:
I applied those rules.